docs/en/operation/Admin-CLI.md

```diff
@@ -60,8 +64,12 @@ It is primarily intended to assist power users in debugging their system.
 
 ```bash
 ./bin/alluxio fsadmin getBlockInfo <block_id>
+BlockInfo{id=16793993216, length=6, locations=[BlockLocation{workerId=8265394007253444396, address=WorkerNetAddress{host=local-mbp, rpcPort=29999, dataPort=29999, webPort=30000, domainSocketPath=, tieredIdentity=TieredIdentity(node=local-mbp, rack=null)}, tierAlias=MEM, mediumType=MEM}]}
+This block belongs to file {id=16810770431, path=/test2}
 ```
 
+### pathConf
```

Comment:
?? What's that?

Reply:
it's a stub for the new path config feature. @cheng-chang needs to write the user docs.